lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_interpolants.jl

```diff
@@ -39,42 +34,37 @@ end
 end
 
 @muladd function _ode_interpolant(Θ, dt, y₀, y₁, k,
-        cache::Union{Rosenbrock23ConstantCache,
-            Rosenbrock32ConstantCache}, idxs::Nothing,
+        cache::Union{RosenbrockConstantCache}, idxs::Nothing,
```

Comment:
remove all the `Union`s where there is only 1 element (style not function)

Reply:
ok. got it.